test/helpers/ssl_test_ctx.c

```diff
@@ -533,6 +533,17 @@ __owur static int parse_expected_key_type(int *ptype, const char *value)
     if (nid == NID_undef)
         nid = EC_curve_nist2nid(value);
 #endif
+    switch (nid) {
+    case NID_brainpoolP256r1tls13:
+        nid = NID_brainpoolP256r1;
+        break;
+    case NID_brainpoolP384r1tls13:
+        nid = NID_brainpoolP384r1;
+        break;
+    case NID_brainpoolP512r1tls13:
+        nid = NID_brainpoolP512r1;
```

Comment:
And what is this good for?

Reply:
This is to handle the 14-curves.cnf tests. Specifically the tests that specify the "ExpectedTmpKeyType". Previously "ExpectedTmpKeyType" expects a name for a curve. But we were passing the name for a TLS group. Until now this has been the same thing. Now we have some TLS groups that use a different name for the underlying curve.